federation/pkg/federation-controller/ingress/ingress_controller.go

```diff
@@ -537,7 +536,7 @@ func (ic *IngressController) getMasterCluster() (master *federation_api.Cluster,
 */
 func (ic *IngressController) updateClusterIngressUIDToMasters(cluster *federation_api.Cluster, fallbackUID string) {
 	masterCluster, masterUID, err := ic.getMasterCluster()
-	clusterObj, clusterErr := conversion.NewCloner().DeepCopy(cluster) // Make a clone so that we don't clobber our input param
+	clusterObj, clusterErr := api.Scheme.DeepCopy(cluster) // Make a clone so that we don't clobber our input param
```

Comment:
Are the ingress deepcopy functions really registered in the global `api.Scheme` object?